src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -367,12 +367,14 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper.setOverrideCatalog(self.overrideCatalog)
         procTaskCmsswHelper.setErrorDestinationStep(stepName=procTaskLogArch.name())
 
-        if forceMerged:
+        if forceMerged and isinstance(forceMerged, bool):
             procTaskStageHelper.setMinMergeSize(0, 0)
-        elif forceUnmerged:
+        elif forceUnmerged and isinstance(forceUnmerged, bool):
```

Comment:
How is that supposed to work ? Boolean or List for forceUnmerged triggers completely different code paths, so I need to check for type at some point...